tutorials/Microphysics/ex_1_saturation_adjustment.jl

```diff
@@ -210,19 +210,27 @@ function main()
     mpicomm = MPI.COMM_WORLD
 
     # output for paraview
+
+    # initialize base prefix directory from rank 0
+    vtkdir = abspath(joinpath(ClimateMachine.Settings.output_dir, "vtk"))
```

Comment:
do we need to check if `Settings.vtk != "never"`? Otherwise what will happen if no argument is used?

Reply:
what should be the behaviour to be in that case?  error out if the output directory is not defined?

Reply:
maybe run the simulation but skip over output generation?

Reply:
ah, i was worried that the output directory wasn't defined by default. I think this looks fine.